frontend/src/views/ManagementPage/index.vue

```diff
@@ -0,0 +1,140 @@
+          <CanteenCard :canteen="canteen" class="fill-height" />
+        </v-col>
+        <v-col cols="12" sm="6" md="4" height="100%">
+          <v-card
+            color="grey lighten-5"
+            class="fill-height d-flex flex-column align-center justify-center"
+            :to="{ name: 'NewCanteen' }"
+          >
+            <v-icon size="100">mdi-plus</v-icon>
+            <v-card-text class="font-weight-bold pt-0 text-center">
+              Ajouter une cantine
+            </v-card-text>
+          </v-card>
+        </v-col>
+      </v-row>
+    </div>
+    <div class="my-12">
+      <h2 class="">Mes diagnostics</h2>
+      <v-btn text color="primary" class="mt-2 mb-8 ml-n4" :to="{ name: 'NewDiagnostic' }">
+        <v-icon class="mr-2">mdi-plus</v-icon>
+        Ajouter un diagnostic
+      </v-btn>
+      <v-row>
+        <v-col cols="12" v-for="diagnostic in diagnostics" :key="`diagnostic-${diagnostic.id}`">
+          <DiagnosticCard :diagnostic="diagnostic" :class="{ 'fill-height': $vuetify.breakpoint.mdAndUp }" />
+        </v-col>
+      </v-row>
+    </div>
+    <div class="my-8">
+      <h2 class="my-4">Mes outils</h2>
+      <v-row>
+        <v-col cols="12" sm="6" md="8" height="100%">
+          <v-card outlined class="d-flex flex-column fill-height pa-2">
+            <v-card-title class="font-weight-bold">Tableur d'aide pour le calcul</v-card-title>
+            <v-card-text>
+              Si vous ne connaissez pas votre part de bio, produits durables, produits issues du commerce équitable,
+              nous vous proposons un outil simple pour les calculer. Sous forme de tableur, remplissez vos achats HT
+              suivant leurs labels et/ou sigles de qualité.
+            </v-card-text>
+            <v-spacer></v-spacer>
+            <v-card-actions class="px-4">
+              <v-spacer></v-spacer>
+              <v-dialog max-width="700" v-model="calculatorModal">
+                <template v-slot:activator="{ on, attrs }">
+                  <v-btn outlined color="primary" v-on="on" v-bind="attrs">
+                    Télécharger notre tableur
+                  </v-btn>
+                </template>
+                <CalculatorResourceModal @closeModal="closeCalculatorModal" />
+              </v-dialog>
+            </v-card-actions>
+          </v-card>
+        </v-col>
+        <v-col cols="12" sm="6" md="4" height="100%">
+          <v-card outlined class="d-flex flex-column fill-height pa-2">
+            <v-card-title class="font-weight-bold">Documentation</v-card-title>
+            <v-card-text>
+              Ressources pour les acteurs et actrices de la restauration collective
+            </v-card-text>
+            <v-spacer></v-spacer>
+            <v-card-actions class="px-4">
+              <v-spacer></v-spacer>
+              <v-btn href="https://ma-cantine-1.gitbook.io/ma-cantine-egalim/" target="_blank" outlined color="primary">
+                Visiter
```

Comment:
Ajoute l'icone `open-in-new` ? Mais je sais pas si c'est nécessaire d'ouvrir ça dans un nouveau onglet, qu'est-ce que tu penses ?

Reply:
:thinking:  Vu que c'est un site à part j'ai une préférence pour l'ouvrir dans un nouvel onglet : ça peut être bizarre pour les utilisateurs de partir sur un site (et donc une UI) complètement différente dans la même fenêtre

Reply:
Ok, alors est-ce que tu es d'acc pour ajouter l'icone ?

Reply:
Yes complètement :+1:

Reply:
Je prends :point_up:

Reply:
Fixé ici : https://github.com/betagouv/ma-cantine/pull/308/commits/9a96cbbf582ff269ede63c0ee53d2eddfdb33633